### PR TITLE
[Docs] Add some notes about native-image flags we use

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

File: docs/src/main/asciidoc/performance-measure.adoc [hunk 1]
```diff
@@ -162,3 +163,51 @@ $ date +"%T.%3N" &&  ./target/quarkus-timing-runner
...
+set `-H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime`.
+
+=== Others ...
+
+This section is provided as high level guidance, but can't presume to be comprehensive as some flags are controlled
+ dynamically by the extensions, the platform you're building on, configuration details, your code and possibly
+ a combination of all these.
+
+Generally speaking the ones listed here are those most likely to affect performance metrics, but in the right
+circumstances one could observe non negligible impact from the other flags too:
```
Comment (gsmet) [inline docs/src/main/asciidoc/performance-measure.adoc]: Really minor but did you really want a colon here at the end of the sentence?

File: docs/src/main/asciidoc/performance-measure.adoc [hunk 2]
```diff
@@ -162,3 +163,51 @@ $ date +"%T.%3N" &&  ./target/quarkus-timing-runner
...
+
+Quarkus extensions can automatically pick the right services they need, while GraalVM's native-image defaults to include
+all services it's able to find on the classpath.
+
+We prefer listing services explicitly as it produces better optimised binaries. Disable it as well by setting `-H:-UseServiceLoaderFeature`.
+
+=== Better default for Garbage Collection implementation
+
+The default in GraalVM seems meant to optimise for short lived processes. Quarkus defaults to server applications, so switch:
+set `-H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/performance-measure.adoc]: I wonder if there's something wrong with `switch: set` or if it's just me?

**Review by gsmet — APPROVED** [review]
@Sanne nice addition. I added 2 comments inline, could you take a look before merging?

File: docs/src/main/asciidoc/performance-measure.adoc (hunk 1 above)
Comment (Sanne) [reply docs/src/main/asciidoc/performance-measure.adoc]: no I didn't. There was a sentence t hat I decided to remove.. Thanks!

File: docs/src/main/asciidoc/performance-measure.adoc (hunk 2 above)
Comment (Sanne) [reply docs/src/main/asciidoc/performance-measure.adoc]: what's wrong? LGTM but I'll rephrase for clarity.

**Comment by Sanne** [comment]
improved base on @gsmet 's feedback, then rebased as well.

**Review by gsmet — APPROVED** [review]

**Pull request merged**